Utils/DialogChanger.cs

```diff
@@ -92,19 +96,75 @@ public static string getDialogMapping(string dialogID)
                 }
             }
             */
+
+            /*
```

Comment:
Was this commented out block left here to potentially use later or is it old code to clean up?

Reply:
This is the original method of handling I was using and ended up finding a better method so yeah it needs to be cleaned up.